tests/autologout.test

```diff
@@ -474,7 +474,7 @@ class AutologoutTestCase extends DrupalWebTestCase {
    *   The test grouping
    */
   public function assertAutotimeout($uid, $expected_timeout, $message = '', $group = '') {
-    return $this->assertEqual(_autologout_get_user_timeout($uid), $expected_timeout, $message, $group);
+    return $this->assertEqual(Drupal::service('autologout.manager')->autologoutGetUserTimeout($uid), $expected_timeout, $message, $group);
```

Comment:
Let's be consistent with the use of `\Drupal` instead of just `Drupal`.
This would anyways not work, since you do not have `use Drupal`.